app/Http/Controllers/Api/ApiController.php

```diff
@@ -64,13 +64,17 @@ public function item($data, $code = 200, $meta = [], $transformer = null)
      * @param  array  $meta
      * @return \Illuminate\Http\JsonResponse
      */
-    public function transform($data, $code = 200, $meta = [])
+    public function transform($data, $code = 200, $meta = [], $include = null)
```

Comment:
i'm not too bothered by it, but you could switch the order of the parameters of you want since you always pass `$include` and don't always need `$meta`and set a default for it - That way you could call `transfom($data, $code, $include)`